browser/src/control/Control.JSDialog.js

```diff
@@ -872,11 +873,11 @@ L.Control.JSDialog = L.Control.extend({
 			target.translateX = newX;
 			target.translateY = newY;
 
-			this.updatePosition(target.container, newX, newY);
+			this.setNewPosition(target.container, newX, newY);
 		}
 	},
 
-	updatePosition: function (target, newX, newY) {
+	updateAutoPopPosition: function (target, newX, newY) {
```

Comment:
I like the new naming concept which out of the box points for what it should be used :)